packages/ui-contract-editor/src/ContractEditor/plugins/withVariables.js

```diff
@@ -57,16 +57,15 @@ export const isEditableVariable = (lockText, editor, event) => {
   const textLength = Node.get(editor, editor.selection.focus.path).text.length;
   const atEnd = editor => textLength === editor.selection.focus.offset;
   const editable = inVariable(editor) && !inReadOnlyVariable(editor);
-
   if (editable) {
     if (atEnd(editor) && event.inputType === 'deleteContentForward') {
       return false;
     }
     if (event.inputType === 'deleteContentBackward') {
       // Do not allow user to delete variable if only 1 char left
-      if (textLength === 1) {
-        return false;
-      }
+      // if (textLength === 1) {
```

Comment:
Please remove the commented code and clean up

Reply:
Sure, will do that.

Reply:
Done.